docs/configuration/secure.md

```diff
@@ -0,0 +1,571 @@
+
+The only files and directories that MUST be owned/writeable by the web user are:
+
+  * ilias.ini.php
+  * data/
+  * ILIAS data dir outside of the webservers docroot
+
+All the other files and directories should be owned by ```root```, but readable by the web user (e.g. 644/755).
+
+example of the suggested configuration:
+```
+git clone --single-branch -b release_5-4 https://github.com/ILIAS-eLearning/ILIAS.git \ /var/www/ilias/.
+
+mkdir -p /var/www/ilias;
+mkdir /var/www/ilias_data;
+mkdir /var/www/ilias_log;
+mkdir /var/www/ilias_log/ilias_errors
+
+chown [www-data:www-data|www_ilias_1:www_ilias_1] /var/www/ilias /var/www/ilias/data
+chown [www-data:www-data|www_ilias_1:www_ilias_1] /var/www/ilias_data
+chown [www-data:www-data|www_ilias_1:www_ilias_1] /var/www/ilias_log
+chown [www-data:www-data|www_ilias_1:www_ilias_1] /var/www/ilias_log/ilias_errors
+
+chmod [2]775 /var/www/ilias/data
+chmod [2]775 /var/www/ilias_data
+chmod [2]775 /var/www/ilias_log
+chmod [2]775 /var/www/ilias_log/ilias_errors
+```
+
+After the installation of ILIAS is finished, you SHOULD also revoke write permission for the file ```ilias.ini.php``` (e.g. ```/var/www/ilias/ilias.ini.php```).
+note: for changing base setting via ILIAS setup, you need to grant write permission for the file ```ilias.ini.php``` again.
+
+## Place data directory outside of the web root
+
+It is highly RECOMMENDED to place your data directory outside of the web server docroot, as pointed out by the ILIAS Installation Wizard.
+
+## Isolate Docroot
+You MAY use openbasedir-restriction to avoid malicious software to directory-traverse out of your docroot-directory. This is very important if there are other websites on the same host.
+
+Apache2:
+
+    php_admin_value open_basedir ./:%docroot_dir%:/usr/share/php/:/var/www/lib/:/tmp
+
+Nginx:
+
+    fastcgi_param PHP_VALUE open_basedir="./:%docroot_dir%:/usr/share/php/:/var/www/lib/:/tmp";
+**Hint:** This option will be applied to the PHP-FPM process. If there are multiple websites on your webserver you have to define a single PHP-FPM-pool for each website. Otherwise these other homepages won' t be accessible anymore.
+
+## OS user handling security
+
+If you use PHP-FPM (FastCGI Process Manager), you can increase security by running the PHP-FPM processes as an specific instead ```www-data``` / ```wwwrun``` (depends on linux distribution).
+
+Snippet from a PHP-FPM pool definition:
+```
+; Unix user/group of processes
+; Note: The user is mandatory. If the group is not set, the default user's group
+;       will be used.
+user = www_ilias_1
+group = www_ilias_1
+```
+
+**note:**  
+NGINX and also apache2 is can only run with one user (no multi user multi process model).  
```

Comment:
Grammar: "is can only" -> "can only"  ;)

Reply:
thanks!